apps/server/src/stores/EventStore.ts

```diff
@@ -20,14 +20,16 @@ export const eventStore = {
   get<T extends keyof RuntimeStore>(key: T) {
     return store[key];
   },
-  set<T extends keyof RuntimeStore>(key: T, value: RuntimeStore[T]) {
+  set<T extends keyof RuntimeStore>(key: T, value: RuntimeStore[T], defere = false) {
     store[key] = value;
     // TODO: Partial updates seems to cause issues on the client
     // socket.send({
     //   type: `ontime-${key}`,
     //   payload: value,
     // });
-    this.broadcast();
+    if (!defere) {
+      this.broadcast();
+    }
```

Comment:
I think adding this optional flags are usually a bad idea.
I dont think we need this and rather build debouncing where necessary. Including here